package.json

```diff
@@ -4,6 +4,9 @@
   "description": "",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "bin": {
+    "iris": "iris.sh"
+  },
   "files": [
     "dist"
```

Comment:
You need to add the `iris.sh` file here for it to be included in the pack.

If you don't do so the cli won't work with globals installations. You can check that is bundling the package the proper way doing `npm run pack` then `tar -xf repositive-iris*` and checking that the package folder has the `rps.sh` in it.